packages/fxa-payments-server/src/store/index.tsx

```diff
@@ -69,134 +69,173 @@ export const selectors: Selectors = {
   },
 };
 
-export const actions: ActionCreators = {
```

Comment:
I had Redux action and thunk creator functions mixed in the same object, here. But, the Redux upgrade came with stricter typescript annotations that seemed to turn this into a mess of errors. Separating the actions & thunks into separate objects seemed to make things happier.

Reply:
FWIW, we also have issue #1079 for generally improving the Redux + Typescript situation here. We should probably revisit this area in that issue and rework it into something cleaner and more sensible.